### ISSUE TITLE
Issue in extracting XML text with parentheses

### ISSUE THREAD

**Issue by khurramlone** [buglabs/node-xml2json#73]
Hi,

The converter works very well in most circumstances, but on extracting XML text with parentheses it returns the HTML code e.g. (test) is returned as  &amp;&#35;40;test&#41;. Is there a solution for this problem?

Thanks

**Comment by c4milo** [comment]
@khurramlone I'm sorry you are having problems with this module, do you have a code snippet that reproduces the problem?

**Comment by radicand** [comment]
```
> x.toJson('<test>(testing)</test>')
'{"test":"&amp;&amp;#35;40;testing&amp;&amp;#35;41;"}'
```

**Comment by c4milo** [comment]
A fix landed in 8504b1bcd92d723f1bb21699488bd7a3401ab541. Version 0.8.1 was released in NPM including it.
